src/averageAges.js

```diff
@@ -34,7 +37,14 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const women = people
+    .filter(item =>
+      item.sex === 'f'
+      && (withChildren === undefined
```

Comment:
explicite is good, you do not must change it, but more convenient just `!withChildren` if you do not need differences between `false`, `null`, `undefined`, etc.